src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -9,45 +9,48 @@ namespace NServiceBus
     using Outbox;
     using Persistence;
     using Pipeline;
+    using Settings;
     using Transport;
     using Unicast;
 
     class ReceiveComponent
     {
-        ReceiveComponent(ReceiveConfiguration configuration,
+        ReceiveComponent(ReceiveConfiguration transportReceiveConfiguration,
             Func<IPushMessages> messagePumpFactory,
             PipelineComponent pipeline,
             IEventAggregator eventAggregator,
             CriticalError criticalError,
-            string errorQueue)
+            string errorQueue) 
         {
-            this.configuration = configuration;
+            this.transportReceiveConfiguration = transportReceiveConfiguration;
             this.messagePumpFactory = messagePumpFactory;
             this.pipeline = pipeline;
             this.eventAggregator = eventAggregator;
             this.criticalError = criticalError;
             this.errorQueue = errorQueue;
         }
 
-        public static ReceiveComponent Initialize(ReceiveConfiguration receiveConfiguration,
+        public static ReceiveComponent Initialize(Configuration configuration,
+            ReceiveConfiguration transportReceiveConfiguration,
```

Comment:
Nitpick: This rename make the PR harder to review, perhaps leaving it as receiveConfiguration would have been better until we can tackle it?

Reply:
PR review in a browser or VS?
I'd rather keep the names proper if that's OK with you...

Reply:
Both, but like a said it was a nitpick

Reply:
I'll do my best next time to make it easier for the review. Maybe break into smaller PRs, where the last commit is the renaming.